grpc/execution/server.go

```diff
@@ -57,6 +58,22 @@ func NewExecutionServiceServerV1Alpha2(eth *eth.Ethereum) *ExecutionServiceServe
 	}
 }
 
+func (s *ExecutionServiceServerV1Alpha2) GetGenesisInfo(ctx context.Context, req *astriaPb.GetGenesisInfoRequest) (*astriaPb.GenesisInfo, error) {
+	log.Info("GetGenesisInfo called", "request", req)
+
+	rollupId := sha256.Sum256([]byte(s.bc.Config().AstriaRollupName))
```

Comment:
nit: Inject the ID into the config as part of startup instead of calculating it ad hoc? No hard opinion on that tho.

Reply:
I think this is the best way to do this without messing with how configs get parsed and such in geth. As many changes kept to this file is better. Especially since this is called once on startup, not a high computation cost.